Anlab.Mvc/Client/components/OrderForm.tsx

```diff
@@ -82,7 +82,7 @@ export default class OrderForm extends React.Component<undefined, IOrderState> {
         this.state = { ...initialState };
     }
     validate = () => {
-        const valid = this.state.quantity > 0 && !!this.state.project.trim();
+        const valid = this.state.quantity > 0 && this.state.quantity <= 100 && !!this.state.project.trim();

```

Comment:
@jSylvestre is there a way we can just ask the quantity box if it is valid, or have a this.state.quantityValid property?  That way we aren't tied in multiple places to this range limit